### DISCUSSION TITLE
🤔根据官网的提示，运行react:playground命令进行调试，报错

### DISCUSSION THREAD

**Discussion by gb853940223** [antvis/S2#2991] [Q&A]
官方文档让使用以下命令进行调试：
![image](https://github.com/user-attachments/assets/540d3908-7348-414d-a127-aca5dbcad189)
我运行这个命令的时候，却报错：
![image](https://github.com/user-attachments/assets/37539a4f-417b-4ba7-b15e-75ba17bd2881)
整个过程严格按照官方流程走的

**Comment by lijinke666** [comment]
看起来你是参考的 `s2-react` 目录下的 README ?

可以参考 [贡献指南](https://s2.antv.antgroup.com/manual/contribution#%EF%B8%8F-%E6%9C%AC%E5%9C%B0%E5%BC%80%E5%8F%91), 看起来你是没有 `pnpm install` ?, 要先让 pnpm 在本地 node_modules 生成软链接
